Mathlib/Algebra/GroupWithZero/Action/Defs.lean

```diff
@@ -3,11 +3,9 @@ Copyright (c) 2018 Chris Hughes. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Chris Hughes, Yury Kudryashov
 -/
-import Mathlib.Algebra.Group.Action.Defs
-import Mathlib.Algebra.Group.Hom.Defs
-import Mathlib.Algebra.Group.TypeTags
-import Mathlib.Algebra.Opposites
-import Mathlib.Logic.Embedding.Basic
+import Mathlib.Algebra.Group.Action.Units
+import Mathlib.Algebra.Group.Equiv.Basic
+import Mathlib.Algebra.GroupWithZero.Units.Basic
```

Comment:
This seems to amount to a 12% increase of the number transitive imports...

Since this is a `Defs` file, I would rather prefer to see the number go down.

Reply:
It is a very recent `Defs` file, however. I am still working on it. Here are my future plans:
1. Move some of the content from `GroupTheory.GroupAction.Group` to this file (this PR)
2. Move most of the content of `Algebra.SMulWithZero` to this file (not yet opened, but you can see the result in #13027)
3. Think about a nice way to split things into `Defs` and `Basic`

I can't really do the last step before the first two, so I would rather stick to my plan and declare this temporary 12% increase as fine.

Reply:
Why can't you do (2) first and then (1) and (3) simultaneously?

Reply:
(2) will increase imports as well